### PR TITLE
add `scaleFactor` option for the PNG/SVG image export action

### LINKED ISSUE #73: add `scaleFactor` as an option for exporting to SVG and PNG
vega-view's `toImageURL()` function accepts a scale factor argument. The scale factor argument would be useful in vega-embed to producer higher resolution PNGs. For SVG, it is probably of lesser importance but would still be nice to have.

The argument should be on conditional on the `export` action. It could be specified either by making `action.export` an object (e.g. `export : { scaleFactor: 5 }`) or by adding a separate `scaleFactor` option. The latter option would likely be less tedious to use.

### PR DESCRIPTION
**Pull request by g3o2**
fix #73 
based on [vega-view](https://github.com/vega/vega-view#view_toImageURL) 's documentation of the `view.toImageURL()` function.

### CODE REVIEW

File: src/embed.ts [hunk 1]
```diff
@@ -225,7 +227,7 @@ export default async function embed(
         .attr('download', `visualization.${ext}`)
         .on('mousedown', function(this: HTMLLinkElement) {
           view
-            .toImageURL(ext)
+            .toImageURL(ext, scaleFactor)
```
Comment (domoritz) [inline src/embed.ts]: You can just pass opt.scaleFactor. it's okay to pass in undefined.

Comment (g3o2) [reply src/embed.ts]: you mean `toImageURL(ext, opt.scaleFactor)` without the default defined at line 103 ? Shall I also re-adjust the docs as well then? (docs mention the default of 1)

Comment (domoritz) [reply src/embed.ts]: The default is still 1 as that's what Vega uses but there is no need to redefine the default here.

**Comment by domoritz** [comment]
Thank you!

**Pull request closed without merging**